timemachine/md/moves.py

```diff
@@ -103,6 +108,31 @@ def n_proposed_by_move(self) -> List[int]:
         return [m._n_proposed for m in self.moves]
 
 
+class Choice(CompoundMove[_State]):
```

Comment:
nit: naming -- `MixtureOfMoves` or similar?

Reply:
Nice, I like `MixtureOfMoves`. Will update.

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1128/commits/543cac9b411362b37946bf85c0c251cc4e7d7711